unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarBodyPartReferenceHandler.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AvatarBodyPartReferenceHandler : MonoBehaviour
+{
+    [HideInInspector]
+    public Transform footL, footR, handL, handR;
+
+    private void Awake()
```

Comment:
I was looking at how this component is used, and I think that is reasonable enough to just make the class an utility class.

We can do something like:
```public static class AvatarBodyPartUtils 
{
    public static void FindLeftToe( Transform[] bodyParts )
    {
       ...
    }
    
    etc...
}
```